src/Utopia/Messaging/Adapters/SMS/ClickSend.php

```diff
@@ -41,6 +41,15 @@ protected function process(SMS $message): string
             fn ($to) => \ltrim($to, '+'),
             $message->getTo()
         );
+        $body = array();
+        foreach ($to as $t) {
+            array_push($body, [
```

Comment:
Let's use modern array syntax here

```suggestion
            $body[] = [
```